### DISCUSSION TITLE
Update Version: 2024.1.7 Issues

### DISCUSSION THREAD

**Discussion by Borgjoy** [matt8707/ha-fusion#256] [General]
Hi there,

I have been using this add-on for a few weeks, it is great thank you.

I have just updated to version 2024.1.7, I immediately noticed I couldn't start the add-on as usual so restarted HA and encountered the same issue. I then noticed that the HA URL had been removed or cleared, I entered my URL instance but am now greeted with a  HA login prompt but the add-ons login button is not responsive/clickable (I enter my valid user name and password). I have tried both with Chrome and Brave and restarting Fusion

Thanks
![Capture](https://github.com/matt8707/ha-fusion/assets/140389435/2496a01d-12f7-4c28-b313-601c50b7f4ba)

**Comment by cokeiill** [comment]
I'm also having the same issue, can't log in,

**Comment by zsw12abc** [comment]
me 2 cant login..

**Comment by scm35768** [comment]
I had the same problem, when you enter the URL in the setup wrong, it is not working. Instead of 
http://homeassistant.local:8123/

use

http://homeassistant.local:8123

then it works for me!

**Reply by Borgjoy** [reply]
Hi @scm35768 thanks, that works far me also
